apps/E2E/src/Menu/pages/MenuPageObject.ts

```diff
@@ -70,57 +80,27 @@ class MenuPageObject extends BasePage {
     );
   }
 
+  /* When menu item 1 is clicked, it increments a counter variable in the test component page, which is displayed in a text box. Incrementing a counter within a label
+   * allows us to test whether the onClick() callback fires for click and keypress inputs across individual test cases without having to close the menu and reset the
+   * label per case (as you see in the button + checkbox tests). This decreases test time and improves performance for this spec. */
   async itemOnClickHasFired(timesFired: number): Promise<boolean> {
     return (await (await this._callbackLabel).getText()).includes(timesFired.toString());
```

Comment:
I think we should compare the strings via "===" instead of includes in case we're firing more onClick callbacks than we expect. So in the event we happen to fire "12" onClick callbacks but we only expect "2", we would flag that as an error. Otherwise, the includes function would return true.